src/servicesHost.ts

```diff
@@ -509,3 +520,45 @@ function populateDependencyGraphs(
     ] = true;
   });
 }
+
+function addCache(servicesHost: typescript.ModuleResolutionHost) {
+  const clearCacheFuncs: (() => void)[] = [];
```

Comment:
could this become a `type` please?  Perhaps `type Action = () => void;`?

Reply:
Do I need to make return type of `createCache` as `type`/`interface` too?

Reply:
No it's fine I think